fms_extras/utils/cache/expandable.py

```diff
@@ -0,0 +1,181 @@
+import dataclasses
+from typing import Dict, List, Optional, Tuple, Union
+
+import torch
+
+from fms_extras.utils.cache import CacheDataLayer, CacheDataWithMetadata, KVCacheManager
+
+
+@dataclasses.dataclass
+class InPlaceCacheDataLayer(CacheDataLayer):
```

Comment:
same here, what is this cache trying to accomplish?

Reply:
Similar comment to the above, this is not being used. We can probably remove it to reduce the scope